### ISSUE TITLE
Unable to type the character "w" in the chat prompt

### ISSUE THREAD

**Issue by Algomorph** [Exafunction/CodeiumJetBrains#33]
The title of this issue sums up the observed & expected behavior.

Here are IDE/Codeium details:
```
CLion 2024.1.4
Build #CL-241.18034.45, built on June 18, 2024
Licensed to Presage Security, Inc / Gregory Kramida
Subscription is active until November 19, 2024.
Runtime version: 17.0.11+1-b1207.24 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 6.5.0-44-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 8192M
Cores: 32
Registry:
  actionSystem.update.actions.warn.dataRules.on.edt=false
  debugger.attach.dialog.enabled=true
  run.processes.with.pty=TRUE
  ide.experimental.ui=true
  ide.slow.operations.assertion=false
  terminal.new.ui=true
Non-Bundled Plugins:
  idea.plugin.protoeditor (241.15989.49)
  org.turbanov.run.configuration.as.action (3.0)
  com.codeium.intellij (1.10.3)
  dev.meanmail.plugin.nginx-intellij-plugin (2024.3)
  A move tab left and right using the keyboard plugin - by momomo.com (12.550)
  com.mikejhill.intellij.movetab (2.1.1)
  ru.adelf.idea.dotenv (2024.1)
  kemoke.net.cssmodule (1.1)
  com.haulmont.rcb (241.14494.140)
  com.google.idea.bazel.clwb (2024.07.02.0.1-api-version-241)
Current Desktop: ubuntu:GNOME
```

**Comment by Algomorph** [comment]
[Screencast from 07-26-2024 05:29:42 PM.webm](https://github.com/user-attachments/assets/ed9090ac-e774-4762-82ce-67bf4f4614b8)

**Comment by rohanphadte** [comment]
We've released 1.10.7 that fixes this issue.

Best,
Rohan
